server/db/Seeder.js

```diff
@@ -1,24 +1,65 @@
 import pg from "pg"
 import path from "path"
+import LineReader from "line-reader"
 import { fileURLToPath } from "url"
 
 const pool = new pg.Pool({
   connectionString: "postgres://postgres:password@localhost:5432/ruff_ryders_db"
 })
 
-const typeQuery =
-  "INSERT INTO pet_types (type, img_url, description) VALUES ('cat','https://clipartix.com/wp-content/uploads/2019/02/black-cat-clipart-2-2019-10.png','Meows a lot.'), ('dog','https://clipartix.com/wp-content/uploads/2016/12/Dogs-dog-and-cat-silhouette-clip-art-free.jpg', 'Barks a lot.'), ('fox', 'https://clipartix.com/wp-content/uploads/2016/03/Red-fox-clip-art-free-clipart-images.jpg','Says, hehehe.');"
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
 
+const adoptablePetsPath = path.join(__dirname, "../../adoptablePets.txt")
 class Seeder {
   static async seedPetTypes() {
+    const typeQuery =
+      "INSERT INTO pet_types (type, img_url, description) VALUES ('cat','https://clipartix.com/wp-content/uploads/2019/02/black-cat-clipart-2-2019-10.png','Meows a lot.'), ('dog','https://clipartix.com/wp-content/uploads/2016/12/Dogs-dog-and-cat-silhouette-clip-art-free.jpg', 'Barks a lot.'), ('fox', 'https://clipartix.com/wp-content/uploads/2016/03/Red-fox-clip-art-free-clipart-images.jpg','Says, hehehe.');"
+
     try {
       const results = await pool.query(typeQuery)
-      pool.end()
     } catch (error) {
       console.error(error)
       pool.end()
     }
   }
+
+  static async seedAdoptablePets() {
+    LineReader.eachLine(adoptablePetsPath, async (line, last, done) => {
+      const [
+        name,
+        img_url,
+        age,
+        vaccination_status,
+        adoption_story,
+        available_for_adoption,
+        pet_type_id
+      ] = line.split(";")
+
+      const queryString =
+        "INSERT INTO adoptable_pets  (name, img_url, age, vaccination_status, adoption_story, available_for_adoption, pet_type_id ) VALUES ($1, $2, $3, $4, $5, $6, $7);"
+
+      try {
+        const result = await pool.query(queryString, [
+          name,
+          img_url,
+          age,
+          vaccination_status,
+          adoption_story,
+          available_for_adoption,
+          pet_type_id
```

Comment:
Ideally we shouldn't be hardcoding associated foreign keys in our data - we want to make sure that the record actually exists in our database that we're trying to associate, and make sure it's the correct record! Take a look back at this example seeder from the one to many clinic to see how you could query the database for the pet type records you're trying to associate here
https://github.com/LaunchAcademy/one-to-many-associations-express-and-postgres-facilitation/blob/main/server/db/Seeder.js